### PR TITLE
Update ens, ecns, wns, tomo, qkc

### PR DESCRIPTION
**Pull request by billPortalNetwork**

### CODE REVIEW

File: src/helper/wns/resolverService.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }
+}
+
+/**
+ * 
+ * @param {*} support 
+ */
+export const getSupportsInterface = async (support) => {
+  try {
+    const content = await resolver.supportsInterface(support);
```
Comment (PhyrexTsai) [inline src/helper/wns/resolverService.js]: We might better using `isSupport`
```javascript
const isSupport = await resolver.supportsInterface(support);
```

File: src/helper/wns/resolverService.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    return content;
```
Comment (PhyrexTsai) [inline src/helper/wns/resolverService.js]:
```javascript
return isSupport
```

File: src/EthereumClassicNameService.js [hunk 3]
```diff
@@ -8,11 +16,15 @@ class EthereumClassicNameService {
...
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```
Comment (PhyrexTsai) [inline src/EthereumClassicNameService.js]: This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`

File: src/EthereumNameService.js [hunk 4]
```diff
@@ -1,18 +1,30 @@
...
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL, this.networkId);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```
Comment (PhyrexTsai) [inline src/EthereumNameService.js]: This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`

File: src/QuarkChainNameService.js [hunk 5]
```diff
@@ -1,18 +1,31 @@
...
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL, this.networkId);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```
Comment (PhyrexTsai) [inline src/QuarkChainNameService.js]: This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`

File: src/TomochainNameService.js [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+  }
+
+  async getResolver(name) {
+    // TODO using web3 to fetch data
+    registryInit(this.restURL, this.networkId);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
+  }
+
+  async getAddress(name, resolverAddr) {
```
Comment (PhyrexTsai) [inline src/TomochainNameService.js]: This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`

File: src/WanchainNameService.js [hunk 7]
```diff
@@ -1,18 +1,31 @@
...
     // TODO using web3 to fetch data
-    console.log(name);
+    registryInit(this.restURL, this.networkId);
+    const resolverAddr = await getResolver(name);
+    return resolverAddr;
   }
 
-  async getAddress(name) {
-    
+  async getAddress(name, resolverAddr) {
```
Comment (PhyrexTsai) [inline src/WanchainNameService.js]: This function can use more easily just fill in `name`, we can get the `resolver` from `getResolver` function and response the `address`

File: src/helper/wns/registryService.js [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+ * @description 
+ * 
+ * Example Usage: 
+ *   entries("foobar");  // 只需搜尋需要註冊的名稱
+ * 
+ * @param {*} name 
+ * @param {*} label
+ * @param {*} owner
+ */
+export const setSubnodeOwner = async (name, label, owner) => {
```
Comment (PhyrexTsai) [inline src/helper/wns/registryService.js]: We can remove unuse functions

File: src/helper/tns/resolverService.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+  }
+}
+
+/**
+ * 
+ * @param {*} support 
+ */
+export const getSupportsInterface = async (support) => {
+  try {
+    const content = await resolver.supportsInterface(support);
```
Comment (PhyrexTsai) [inline src/helper/tns/resolverService.js]:
```javascript
const isSupport = await resolver.supportsInterface(support);
return isSupport;
```

File: src/helper/tns/registryService.js [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+ * @description 
+ * 
+ * Example Usage: 
+ *   entries("foobar");  // 只需搜尋需要註冊的名稱
+ * 
+ * @param {*} name 
+ * @param {*} label
+ * @param {*} owner
+ */
+export const setSubnodeOwner = async (name, label, owner) => {
```
Comment (PhyrexTsai) [inline src/helper/tns/registryService.js]: Remove unuse function

**Comment by billPortalNetwork** [comment]
Update #1 #2 #3 #5 #6

**Pull request merged**